openApiDocumentation.json

```diff
@@ -206,6 +206,29 @@
           }
         }
       }
+    },
+    "/user/:user_id": {
+      "get": {
+        "parameters": [
+          {
+            "description": "The provider for the connected user",
+            "name": "provider",
+            "in": "query",
+            "required": true,
+            "schema": {
+              "type": "string"
```

Comment:
Looks like I forgot to do this for the data endpoints, but ideally we would put the enum so they can view the options for provider: https://github.com/Universal-Connect-Project/ucw-app/blob/c0755118ec738a6c85d6be5f40d50e335354d4fd/openApiDocumentation.json#L68